core/store/migrate/migrations/0232_log_poller_word_topic_indexes.sql

```diff
@@ -0,0 +1,62 @@
+-- +goose Up
+
+drop index if exists evm.evm_logs_idx_data_word_one;
+drop index if exists evm.evm_logs_idx_data_word_two;
+drop index if exists evm.evm_logs_idx_data_word_three;
+drop index if exists evm.evm_logs_idx_data_word_four;
+drop index if exists evm.evm_logs_idx_topic_two;
+drop index if exists evm.evm_logs_idx_topic_three;
+drop index if exists evm.evm_logs_idx_topic_four;
+
+create index evm_logs_idx_data_word_one
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 1, 32));
+
+create index evm_logs_idx_data_word_two
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 33, 32));
+
+create index evm_logs_idx_data_word_three
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 65, 32));
+
+create index evm_logs_idx_data_word_four
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 97, 32));
+
+create index evm_logs_idx_data_word_five
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 129, 32));
+
+create index evm_logs_idx_topic_two
+    on evm.logs (evm_chain_id, address, event_sig, (topics[2]));
+
+create index evm_logs_idx_topic_three
+    on evm.logs (evm_chain_id, address, event_sig, (topics[3]));
+
+create index evm_logs_idx_topic_four
+    on evm.logs (evm_chain_id, address, event_sig, (topics[4]));
+
+-- +goose Down
+
+drop index if exists evm.evm_logs_idx_data_word_three;
+drop index if exists evm.evm_logs_idx_data_word_five;
+drop index if exists evm.evm_logs_idx_topic_two;
+drop index if exists evm.evm_logs_idx_topic_three;
+drop index if exists evm.evm_logs_idx_topic_four;
+
+create index evm_logs_idx_data_word_one
```

Comment:
Same here, should be `evm.evm...`

Reply:
Oh, maybe this was intentional... since the `evm.evm` is redundant.  In that case, I think the only problem is just below here, where you create the original indices... they have to match what's dropped during `goose up` or the whole thing can become inconsistent